ci/Jenkinsfile.linux

```diff
@@ -0,0 +1,72 @@
+pipeline {
+  agent {
+    docker { 
+      label 'linux'
+      image 'statusteam/nim-status-client-build:latest'
+      /* allows jenkins use cat and mounts '/dev/fuse' for linuxdeployqt */
+      args '--entrypoint="" --cap-add SYS_ADMIN --security-opt apparmor:unconfined --device /dev/fuse'
+    }
+  }
+
+  options {
+    timestamps()
+    /* Prevent Jenkins jobs from running forever */
+    timeout(time: 40, unit: 'MINUTES')
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  environment {
+    /* Improve make performance */
+    MAKEFLAGS = '-j4'
```

Comment:
I think Linux machines have 6 CPU cores while the macOS ones have 12. How about using `nproc`?

Reply:
I see no difference in performance between 4 or 12. It takes the same amount of time.
There's not enough targets to actually take advantage of that. And I can't call commands in `environment` block.

Reply:
> There's not enough targets to actually take advantage of that.

Yes, that's true most of the time. Bootstrapping the Nim compiler from C files can make use of more cores, but you only do that once per compiler version per Jenkins node.

>  can't call commands in environment block

https://github.com/status-im/nim-beacon-chain/blob/c43bf48de29ed314c813d01337fdff342cd6b9aa/Jenkinsfile#L72